docs/usage.md

```diff
@@ -425,6 +425,50 @@ field_name_mapping = {
 ```
 
 
+### Working with polymorphic resources
+
+#### Writing polymorphic resources
+
+A polymorphic endpoint can be setup if associated with a polymorphic serializer.
```

Comment:
"setup" (a noun) should be "set up" (a verb phrase) here. As a native English speaker, I apologize for the stupidity and craziness of the English language. 😄